e2e/commands/generate.test.js

```diff
@@ -0,0 +1,28 @@
+const test = require('ava')
+const fs = require('fs')
+const path = require('path')
+
+const {run} = require('../helpers/test-utils.js')
+
+const genPath = path.join(__dirname, 'generate-cmd')
+const docsPath = path.join(genPath, 'docs')
+
+test.before('create temp directory', () => {
+  // Cleanup if the directory already exists
+  if (fs.existsSync(genPath)) {
+    fs.rmdirSync(genPath, {recursive: true})
+  }
+
+  fs.mkdirSync(genPath)
+})
+
+test.after('cleanup', () => {
+  fs.rmdirSync(genPath, {recursive: true})
+})
+
+test('generate _sidebar.md', t => {
```

Comment:
It would be nice to have a test case for the case in which the respective file already exists.

Reply:
done.